ansible-firewalld/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# This example uses Ansible to configure firewalld to set up a node-local firewall suitable as recommended for use as an OpenShift 4 worker.
+# However, this is intended to generalize to using Ansible as well as firewalld for generic tasks.
+FROM quay.io/coreos-assembler/fcos:testing-devel
+
+# Install firewalld and ansible
+RUN rpm-ostree install firewalld ansible
+
+ADD configure-firewall-playbook.yml .
+
+# Run playbook setting ports.
+RUN ansible-playbook -vvv configure-firewall-playbook.yml
+
+#TO-DO remove ansible when supported by rpm-ostree in container.
```

Comment:
Note this is supported. It'd be `rpm-ostree override remove ansible`. I suspect you're hitting the bug fixed by this patch: [coreos/rpm-ostree@`2b2cc8d` (#3671)](https://github.com/coreos/rpm-ostree/pull/3671/commits/2b2cc8dfba1fb0d7a45c1d4d19358d63b5c6cd9d).

Reply:
Yeah I get:
```
STEP 5/5: RUN rpm-ostree override remove ansible
error: This system was not booted via libostree.
Currently, most rpm-ostree commands only work on ostree-based host systems.
```
Once your patch gets to testing-devel I'll update it to remove it, thanks for fixing this.